main.tf

```
@@ -25,8 +25,9 @@ locals {
   dd_api_key_asm         = local.dd_api_key_resource == "asm" ? { DD_API_KEY_SECRET_ARN = local.dd_api_key_identifier } : {}
   dd_api_key_ssm         = local.dd_api_key_resource == "ssm" ? { DD_API_KEY_SSM_NAME = local.dd_api_key_identifier } : {}
   dd_site                = { DD_SITE = var.forwarder_lambda_datadog_host }
+  dd_tags                = { DD_TAGS = join(",", var.dd_tags) }
```

```suggestion
  dd_tags                = { DD_TAGS = join(" ", var.dd_tags) }
```

https://docs.datadoghq.com/agent/docker/?tab=standard

```


DD_TAGS | Host tags separated by spaces. For example: simple-tag-0 tag-key-1:tag-value-1

```

---

This is what I instinctively put until I double-checked the forwarder docs. See the linked reference:

https://github.com/DataDog/datadog-serverless-functions/blob/060c54c5ae285474d3d25c563ba040b2369e97a3/aws/logs_monitoring/settings.py#L97-L101

> ## @param DD_TAGS - list of comma separated strings - optional -default: none
> ## Pass custom tags as environment variable or through this variable.
> ## Ensure your tags are a comma separated list of strings with no trailing comma in the envvar!

---

the code of the forwarder is not consistent with anything else in Datadog, it is pretty bad

---

it's not easy to understand which doc/code is correct.
@bendrucker  I suppose you tested it, so I'll approve.
Please open another PR if any fixes will be required.